packages/js-example-selected-features/webpack.config.js

```
@@ -0,0 +1,45 @@
+// Generated using webpack-cli http://github.com/webpack-cli
+const path = require('path');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+// hack to get the webpack mode
+const isDevMode = !process.argv.includes('--mode=production');
```

ESLint

[error] 'process' is not defined.

(no-undef)